### PR TITLE
Fix #4 - Implement LIC3

### LINKED ISSUE #4: LIC3
There exists at least one set of three consecutive data points that are the vertices of a triangle with area greater than AREA1.
(0 ≤ AREA1)

### PR DESCRIPTION
**Pull request by MrLarssonJr**

### CODE REVIEW

**Comment by MrLarssonJr** [comment]
Forced-pushed to rebase on top of master.

File: src/test/java/TestLIC3.java [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+        );
+        Point[] pts = {
+                new Point(10, 5),
+                new Point(0, 0),    //-------
+                new Point(10, 0),   // these form an triangle with area 50
+                new Point(0, 10),   //-------
+                new Point(6,6)
+        };
+
+        try {
```
Comment (marcusgranstroem) [inline src/test/java/TestLIC3.java]: This test is redundant, should verify that exception is thrown :)

File: src/test/java/TestLIC3.java [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+            Decide.LIC3(pts, params);  //Should thrown exception
+        } catch(IllegalArgumentException e) {
```
Comment (marcusgranstroem) [inline src/test/java/TestLIC3.java]: This test is redundant, should verify that exception is thrown :)

File: src/test/java/TestLIC3.java [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+                0,
+                0,
+                0,
+                0,
+                0
+        );
+        Point[] pts = {
+                new Point(10, 5),
+                new Point(0, 0),    //-------
+                new Point(10, 0),   // these form an triangle with area 50
```
Comment (helenaalinder) [inline src/test/java/TestLIC3.java]: a triangle*

File: src/test/java/TestLIC3.java [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+        boolean actual = Decide.LIC3(pts, params);
+
+        Assert.assertFalse(actual);
+    }
+
+    /*
+     * Test what happens when 0<=AREA1 isn't true.
+     */
+    @Test
```
Comment (helenaalinder) [inline src/test/java/TestLIC3.java]: Add (expected = ArithmeticException.class) after @Test

File: src/test/java/TestLIC3.java [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+        Point[] pts = {
+                new Point(10, 5),
+                new Point(0, 0),    //-------
+                new Point(10, 0),   // these form an triangle with area 50
+                new Point(0, 10),   //-------
+                new Point(6,6)
+        };
+
+        try {
+            Decide.LIC3(pts, params);  //Should thrown exception
```
Comment (vronc) [inline src/test/java/TestLIC3.java]: fix typo
```suggestion
            Decide.LIC3(pts, params);  //Should throw exception
```

**Pull request merged**